src/components/light.js

```diff
@@ -81,6 +95,21 @@ module.exports.Component = registerComponent('light', {
             break;
           }
 
+          case 'castShadow':
+          case 'shadowBias':
+          case 'shadowCameraFar':
+          case 'shadowCameraFov':
+          case 'shadowCameraNear':
+          case 'shadowCameraTop':
+          case 'shadowCameraRight':
+          case 'shadowCameraBottom':
+          case 'shadowCameraLeft':
+          case 'shadowDarkness':
+          case 'shadowMapHeight':
+          case 'shadowMapWidth':
+            this.updateShadow();
+            break;
```

Comment:
Hm this gets executed repeatedly. I'll try to think of a better way to do this part.